mrblib/common.rb

```diff
@@ -658,8 +658,9 @@ def self.encode_www_form(enum)
     enum.map do |k, v|
       if v.nil?
         encode_www_form_component(k)
-      elsif v.respond_to?(:to_ary)
-        v.to_ary.map do |w|
+      elsif v.respond_to?(:to_ary) or v.is_a?(Array)
+        vv = v.is_a?(Array) ? v : v.to_ary
+        vv.map do |w|
```

Comment:
IMO using `Kernel#Array` is shorter and prettier, which exists in `mruby-kernel-ext` for mruby.

https://github.com/mruby/mruby/blob/9516731329a9d43b4aab57a19fb9cfc1b62f11e8/mrbgems/mruby-kernel-ext/src/kernel.c#L160-L181

Reply:
BTW `to_ary` is removed in 2.0